linked_lists/linked_list/linked_list.py

```diff
@@ -11,7 +11,7 @@ def __str__(self):
 class LinkedList(object):
 
     def __init__(self, head=None):
-        self.head = head
+        self.head = Node(None)
```

Comment:
Delete the line `self.head = Node(node)`.

This line created a Node(None) head.